backend/tests/unit/api/test_10_transformation_api.py

```diff
@@ -1,41 +0,0 @@
-from infrahub.core.constants import InfrahubKind
-from infrahub.core.manager import NodeManager
-from infrahub.core.node import Node
-from infrahub.database import InfrahubDatabase
-from infrahub.message_bus.messages.transform_python_data import TransformPythonDataResponse
-
-
-async def test_transform_endpoint(
-    db: InfrahubDatabase, client, client_headers, default_branch, rpc_bus, register_core_models_schema, car_person_data
-):
-    repositories = await NodeManager.query(db=db, schema=InfrahubKind.REPOSITORY)
-    queries = await NodeManager.query(db=db, schema=InfrahubKind.GRAPHQLQUERY)
-
-    t1 = await Node.init(db=db, schema="CoreTransformPython")
-    await t1.new(
-        db=db,
-        name="transform01",
-        query=str(queries[0].id),
-        repository=str(repositories[0].id),
-        file_path="transform01.py",
-        class_name="Transform01",
-    )
-    await t1.save(db=db)
-
-    # Must execute in a with block to execute the startup/shutdown events
-    with client:
-        mock_response = TransformPythonDataResponse(
-            data={"transformed_data": {"KEY1": "value1", "KEY2": "value2"}},
-        )
-        rpc_bus.add_mock_reply(response=mock_response)
-
-        response = client.get(
-            "/api/transform/python/transform01",
-            headers=client_headers,
-        )
-
-    assert response.status_code == 200
-    assert response.json() is not None
-    result = response.json()
-
-    assert result == {"KEY1": "value1", "KEY2": "value2"}
```

Comment:
I don't think we have another test that's validating this endpoint? Looking at what we had above I don't think it provides a lot of value with the mocked data being returned as we provide it as part of the input. We could add a transform to the internal "car-dealership" repo and test it in the same area as we have the TestProposedChangePipelineRepository test class.

Reply:
Good catch, I've added a new test for this endpoint